files/scripts/rnaseq_pipeline/script4.nf

```diff
@@ -55,9 +54,9 @@ process QUANT {
 }
 
 workflow {
-    Channel.fromFilePairs( params.reads, checkIfExists:true )
-        .set { read_pairs_ch }
+   read_pairs_ch = Channel.fromFilePairs( params.reads, checkIfExists:true )
+   transcriptome_ch = Channel.fromPath( params.transcriptome, checkIfExists:true )
```

Comment:
If you want to make it a value channel, do:
```suggestion
   transcriptome_ch = file( params.transcriptome, checkIfExists:true )
```
Then you do not need the `each` qualifier.

Reply:
I didn't know you could input a file into a value channel. Your solution makes more sense.